addons/escoria-core/game/core-scripts/log/esc_logger.gd

```diff
@@ -262,17 +262,29 @@ func report_errors(p_path: String, errors: Array) -> void:
 # * message: Message to log
 # * err: if true, write in stderr
 func _log(message:String, err: bool = false):
-	var info = OS.get_datetime()
-	info["message"] = message
-	message = "{year}-{month}-{day}T{hour}{minute}{second} {message}" \
-			.format(info)
+	message = "{0} {1}".format([_formatted_date(), message])
+
 	if err:
 		printerr(message)
 	else:
 		print(message)
 	_write_logfile(message)
 
 
+# Returns the current date/time, as a string 
+# formatted for logging. E.g. 2022-04-19T16:10:39
+func _formatted_date() -> String:
+	var info = OS.get_datetime()
+	info["year"] 	= "%04d" % info["year"]
```

Comment:
Please remove tabs in the middle of lines.

Reply:
https://stackoverflow.com/questions/101958/code-formatting-is-lining-up-similar-lines-ok

Reply:
Can you please add some context to your response rather than just a URL? @jeancallisti

Reply:
I applied the requested fix anyways, so let's just move forward